src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/Microservice.kt

```diff
@@ -67,7 +66,7 @@ class Microservice(
     private val rootEventId = rootEventId ?: eventRouter.storeEvent("Dirty TCP client".toEvent()).id
     private val errorEventId by lazy { eventRouter.storeEvent("Errors".toErrorEvent(), rootEventId).id }
 
-    private val executor = Executors.newScheduledThreadPool(settings.totalThreads).apply {
+    private val executor = Executors.newScheduledThreadPool(settings.appThreads + settings.ioThreads).apply {
```

Comment:
Maybe use different executors for app and IO tasks? Also, we can choose different priorities for threads

Reply:
In this case, let's stay only one option in the settings to manage executor size.

Reply:
But how we would know much threads to use for IO?